docs/contributing/README.md

```diff
@@ -1,9 +1,142 @@
-We'd love you to join us!
+Knative is an open source project with an active development community.
+The project was started by Google but has contributions from a growing number of industry-leading companies. With a clear project scope, lightweight governance model, and clean lines of
+separation between pluggable components, the Knative project establishes an
+efficient contributor workflow.
 
-In this section:
+Knative is a diverse, open, and inclusive community.
 
-- [Contribute to Knative](contributing.md): helpful tips for how to get started contributing to Knative.
-- [About the Knative community](about.md): information about the Knative community and how it's run.
+Knative is designed for different personas:
+
+![Diagram that displays different Audiences for Knative](images/knative-audience.svg)
+
+Knative components offer developers Kubernetes-native APIs for deploying
+serverless-style functions, applications, and containers to an auto-scaling
+runtime.
+
+To join the conversation, head over to the
+[Knative users](https://groups.google.com/d/forum/knative-users) Google group.
+
+Knative components are intended to be integrated into more polished products
+that cloud service providers or in-house teams in large enterprises can then
+operate.
+
+Any enterprise or cloud provider can adopt Knative components into their own
+systems and pass the benefits along to their customers.
+
+## Community Meetup
+
+This virtual event is designed for end users, a space for our community to meet,
+get to know each other, and learn about uses and applications of Knative.
+
+Catch up with past community meetups on our [YouTube channel](https://www.youtube.com/playlist?list=PLQjzPfIiEQLLyCyLBKLlwDLfE_A-P7nyg).
+
+Stay tuned for new events by subscribing to the
+[calendar](https://calendar.google.com/calendar/embed?src=knative.team_9q83bg07qs5b9rrslp5jor4l6s%40group.calendar.google.com&ctz=America%2FLos_Angeles) ([iCal export file](https://calendar.google.com/calendar/ical/knative.team_9q83bg07qs5b9rrslp5jor4l6s%40group.calendar.google.com/public/basic.ics))
+and following us on [Twitter](https://twitter.com/KnativeProject).
+
+## More ways to get involved
+
+Even if there’s not an issue opened for it, we can always use more testing throughout the platform. Similarly, we can always use more docs, richer docs, insightful docs. Or maybe a cool blog post? And if you’re a web developer, we could use your help in spiffing up our public-facing web site.
+
+[Bug reports](https://github.com/knative/serving/issues/new) and friction logs
+from new developers are especially welcome.
+
+## Knative authors
+
+For a current list of the authors, see [Authors](https://github.com/knative/serving/blob/main/AUTHORS).
+
+## Getting started
+
+Before you can make a contribution to Knative, you must sign the CNCF [EasyCLA](https://easycla.lfx.linuxfoundation.org/) using the same email address you used to register for Github.
```

Comment:
```suggestion
Before you can make a contribution to Knative, you must sign the CNCF Contributor License Agreement, [EasyCLA](https://easycla.lfx.linuxfoundation.org/) using the same email address you use on Github.
```
It might be good to spell out what CLA stands for. 

The Github change is because what you really need is the email address used when doing `git commit` on your code contributions. This typically aligns with an email address used on GitHub but may not align with the one used when registering for an account.

Reply:
Added this into the section in `contributing.md` covering the CLA. Since not everyone participating in the community is necessarily making code contributions, I reverted the move of the CLA documentation into the main README.